core/kazoo_documents/src/kzd_comments.erl

```diff
@@ -26,7 +26,7 @@ new() ->
 
 -spec comments(doc()) -> kz_term:api_objects().
 comments(Doc) ->
-    comments(Doc, 'undefined').
+    comments(Doc, []).
```

Comment:
Update the spec to return `kz_json:objects()` here.

Reply:
Fixed the spec.